app/views/exercises/show.html.haml

```diff
@@ -20,9 +20,16 @@
   #wrapper<
     %pre= render "shared/code/#{name_of_partial}"
   %footer
-    .help Hint
-    .prev= link_to "Prev", exercise_path(prev_exercise)
-    .next= link_to "Next", exercise_path(next_exercise)
+    .hint Hint
+    .submit= link_to "Submit", "#",  data: {next_exercise: next_exercise.id}
+    #message
+      #success
+        %p Nice refactoring!
+        #next= link_to "Go to next exercise", exercise_path(next_exercise)
+      #fail
+        %p Ups... review your code, something is wrong
```

Comment:
Ups sounds like a German way of saying it :trollface: It's also a big delivery company in the US. Not a big deal at all, in the US we tend to use "Oops" instead.